ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -287,13 +296,24 @@ export class FrontstageDef {
 
     const oldState = this._nineZoneState;
     this._nineZoneState = state;
-    this.handlePopouts(oldState);
+    const popoutResult = this.handlePopouts(oldState);
     if (oldState) {
       this.triggerStateChangeEvents(oldState);
     }
 
     const isClosing = this._isStageClosing || this._isApplicationClosing;
     if (isClosing && !this.isReady) return;
+    if (popoutResult === false) {
```

Comment:
We are not emitting an event, but we are not resetting the state either. `frontstageDef.nineZoneState` will still contain a popout widget.

Reply:
Looking into further

Reply:
The state is being reset in FronstageDef.tsx line 873. The state appears fine to me when debugging whenever a popout fails. The widget is contained as floating and not as a popout